system/web/services/RoutingService.cfc

```diff
@@ -1099,17 +1099,12 @@ component extends="coldbox.system.web.services.BaseService" accessors="true" {
 			results[ "scriptName" ] = replaceNoCase( results[ "scriptName" ], getContextRoot(), "" );
 		}
 
-		// Clean up the path_info from index.cfm
-		results[ "pathInfo" ] = reReplaceNoCase( results[ "pathInfo" ], "^[/\\]index\.cfm", "" );
-
-		// Clean the scriptname from the pathinfo if it is the first item in case this is a nested application
-		if ( len( results[ "scriptName" ] ) ) {
-			results[ "pathInfo" ] = reReplaceNoCase(
-				results[ "pathInfo" ],
-				"^#results[ "scriptName" ]#\/",
-				""
-			);
-		}
+		// Clean up the from script name path_info
```

Comment:
Few questions.  
- Why use `reReplace` if the script name is not a regex?
- Why not use the cleaned `script_name` via `results[ "script_name" ]`? Instead of the raw value

Reply:
@lmajano 

> Why use reReplace if the script name is not a regex?

That's your code man, lol.  The rereplace was already there.  I just changed its inputs.  You'll need to ask yourself why you used a regex-based replacement but I just kept it the way you had it.  In reality, you are wrong that's not a regex.  Look at the code-- you're making it a regex like so:
```
"^#results[ "scriptName" ]#\/",
```
by adding `^` to it so it only matches at the start of the string.  Now, that said-- this probably should be escaping any regex metacharacters which may appear in the script name as well (such as periods), but that's an existing issue with the code. 

> Why not use the cleaned script_name via results[ "script_name" ]? Instead of the raw value

Because I didn't trust it :)  `results[ "script_name" ]` is the result of some questionable find/replace above and I can't trust that to not have screwed with the value.  You're doing some sort of odd replacement of index.cfm inside the script name so I can't use that value.  I need the original, actual, real, raw value that came from the web server for this logic to work.